### PR TITLE
fix: flaky tests

### PR DESCRIPTION
**Pull request by neel-ds**
- Delete get-unopened-mysteryBox action (stale)
- Delete process-pending-tx cron as Credit Release has passed one week
- Fix OpenMysteryBox with proper error handling
- Fix get-stack test timezone issue (only UTC)

### CODE REVIEW

**Review by samott — DISMISSED** [review]

File: app/actions/mysteryBox/open.ts [hunk 1]
```diff
@@ -153,70 +161,78 @@ export async function openMysteryBox(
...
             },
-          });
+          );
         },
         {
-          isolationLevel: "Serializable",
-          timeout: ONE_MINUTE_IN_MILLISECONDS,
+          retries: 2,
+          onFailedAttempt: (error) => {
+            console.log("Retrying", error);
```
Comment (marvinmarnold) [inline app/actions/mysteryBox/open.ts:222]: This log could be more descriptive.

File: app/actions/mysteryBox/open.ts [hunk 2]
```diff
@@ -153,70 +161,78 @@ export async function openMysteryBox(
...
-                userId: payload.sub,
-                mysteryBoxPrizeId: prize.id,
-              },
-            });
-          }
+      await pRetry(
+        async () => {
+          await prisma.$transaction(
+            async (tx) => {
+              const txDate = new Date();
```
Comment (marvinmarnold) [inline app/actions/mysteryBox/open.ts:168]: I'm nervous whenever we change the logic for reveals. I also don't really understand all these changes.

Can you add some tests that cover all of the edge cases you have contemplated?

**Review by marvinmarnold — CHANGES_REQUESTED** [review]

File: app/actions/mysteryBox/open.ts [hunk 3]
```diff
@@ -153,70 +161,75 @@ export async function openMysteryBox(
...
-    release();
+      await prisma.mysteryBox.update({
+        where: {
+          id: mysteryBoxId,
+        },
+        data: {
+          status: EMysteryBoxStatus.Opened,
+        },
+      });
+    }
```
[the remark this answers is not in the document]
Comment (neel-ds) [reply app/actions/mysteryBox/open.ts:223]: Oh! you're right. fixed it

**Review by marvinmarnold — APPROVED** [review]

**Review by samott — APPROVED** [review]
